### PR TITLE
Alpakka 3.0.1 & Akka 2.6.14

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

**Comment by ennru** [comment]
The Alpakka Elasticsearch API changed, so those parts fail to build.

File: alpakka-sample-mqtt-to-kafka/project/Dependencies.scala [hunk 1]
```diff
@@ -3,8 +3,8 @@ import sbt._
 object Dependencies {
   val scalaVer = "2.13.2"
   // #deps
-  val AkkaVersion = "2.6.4"
-  val AlpakkaVersion = "2.0.0"
+  val AkkaVersion = "2.6.14"
+  val AlpakkaVersion = "3.0.1"
   val AlpakkaKafkaVersion = "2.0.5"
   val JacksonDatabindVersion = "2.10.3"
```
Comment (marcospereira) [inline alpakka-sample-mqtt-to-kafka/project/Dependencies.scala:9]: Maybe you want to update Jackson to be consistent with the version used by Alpakka 3. See https://github.com/akka/alpakka/pull/2653.

File: alpakka-sample-sqs-java/build.sbt [hunk 2]
```diff
@@ -3,16 +3,16 @@ name := "alpakka-samples-sqs-java"
 
 ThisBuild / scalaVersion := "2.13.3"
 
-val akkaVersion = "2.6.9"
-val alpakkaVersion = "2.0.1"
+val AkkaVersion = "2.6.14"
+val AlpakkaVersion = "3.0.1"
 val jacksonVersion = "2.10.5"
```
Comment (marcospereira) [inline alpakka-sample-sqs-java/build.sbt:8]: Same comment about Jackson version as above.

**Review by marcospereira — COMMENTED** [review]
Just a minor comment about the Jackson version. Besides that, LGTM.

**Pull request merged**